src/components/gallery/Options.jsx

```diff
@@ -1,5 +1,41 @@
+import React from "react";
+import Button from "../Button";
 const Options = () => {
-  return <div>Options</div>;
+  return (
+    <div className="flex flex-col items-center ">
+      <div className="mb-2">
+        <Button
```

Comment:
the button component is separate from this, use regular html \<button\> and add the background color to that

right now there is no background color

<img width="234" alt="image" src="https://github.com/acm-ucr/rishi-website/assets/90938120/e3f686e4-cc30-4102-ae45-08c286587e2d">

Reply:
also add your issue under development
<img width="319" alt="image" src="https://github.com/acm-ucr/rishi-website/assets/90938120/343cdd4b-769e-4d41-a8cf-7c8644c73140">